### PR TITLE
chore: [NMP-95] CHORE** added documentation for readme for frontend

### PR DESCRIPTION
**Pull request by kcaparas1630**
# Description

Added instructions in Readme for future contributors, and co-workers

## Type of changes

Please delete options that are not relevant.

- [x] Readme

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-sustainment-capstone-2024-96-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-sustainment-capstone-2024-96-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/merge.yml)

### CODE REVIEW

**Review by LocalNewsTV — APPROVED** [review]

**Review by GDamaso — APPROVED** [review]

File: frontend/README.md [hunk 1]
```diff
@@ -1,36 +1,97 @@
...
+Install dependencies
+
+```bash
+  npm install
+```
+
+Start docker compose
+
+```bash
+  docker-compose up
```
Comment (GDamaso) [inline frontend/README.md:42]: Don't you need the `--watch` flag here to get hot reload?

**Review by GDamaso — APPROVED** [review]

**Comment by GDamaso** [comment]
Good Work mate! Frontend Readme is a lot nicer and useful now! :rocket:

File: frontend/README.md (hunk 1 above)
Comment (LocalNewsTV) [reply frontend/README.md:42]: The line below covers that @GDamaso

**Comment by DerekRoberts** [comment]
Force merging after group discussion.  Fixes are ongoing.

**Pull request merged**